service/messageHandler_test.go

```diff
@@ -213,32 +204,23 @@ func TestMessageProducerError(t *testing.T) {
 		}`, contentUUID),
 	}
 
-	err := service.HandleMessage(inbound)
-	assert.EqualError(t, err, errmsg)
+	service.HandleMessage(inbound)
 
 	mp.AssertExpectations(t)
-
-	// check of monitoring logging
-	logTest.Assert(t, hook.LastEntry()).
-		HasMonitoringEvent("Map", testTxID, "Annotations").
-		HasValidFlag(true).
-		HasUUID(contentUUID)
+	assert.Equal(t, errmsg, mp.err)
 }
 
 func TestNilWhitelistIsIgnoredWithErrorLog(t *testing.T) {
-	hook := logTest.NewTestHook("test")
+	log := logger.NewUnstructuredLogger()
 	mp := &mockMessageProducer{}
-	service := NewAnnotationMapperService(nil, mp)
+	service := NewAnnotationMapperService(nil, mp, log)
 	inbound := kafka.FTMessage{
 		Headers: map[string]string{"Origin-System-Id": testSystemID},
 		Body:    `{"foo":"bar"}`,
 	}
 
-	actual := service.HandleMessage(inbound)
-	assert.NoError(t, actual)
+	service.HandleMessage(inbound)
 	mp.AssertExpectations(t)
 
-	actualLog := hook.LastEntry()
-	assert.Equal(t, "Skipping this message because the whitelist is invalid.", actualLog.Message, "log message")
```

Comment:
Just wondering if there's something concerning about the hook assertions that we've removed?

Reply:
Well they rely on the old logic of the logger where we don't pass a pointer to a logger instance, but instead the logger is initialised in main.go and used everywhere directly from the library, so `hook.LastEntry()` will return nil on all the tests.